Sources/Microcharts.Uwp/ChartView.cs

```diff
@@ -1,22 +1,41 @@
-﻿// Copyright (c) Aloïs DENIEL. All rights reserved.
+// Copyright (c) Aloïs DENIEL. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+#if !MICROCHARTS_UWP && !MICROCHARTS_WINUI
+#error Must define exactly one of the required symbols to use this shared file
+#endif
+
+#if MICROCHARTS_UWP
 namespace Microcharts.Uwp
+#elif MICROCHARTS_WINUI
+namespace Microcharts.WinUI
+#endif
```

Comment:
```suggestion
namespace Microcharts.WinUI
```

Should use the same namespace for both platforms so XAML building on top can be shared between libraries/apps that are using both technologies or are migrating. See comments in #326 as well.

Reply:
Looks like the WinUI effort is being duplicated anyway and that should be resolved. Personally I prefer that logically different types should have fully different names (i.e. different "namespace.name") because that way they're a bit more searchable, documentable, etc. It's not too hard to imagine that these types could diverge more in the future because the systems they build on are intrinsically incompatible. I get that it makes some things easier when they're the same name, but I always have a hard time with different things having the same name.

Also, the breaking change concern could be a real one (though I don't know how popular the UWP one already is).

But, not my project, so up to the owner(s) to make the call on this 😄

Reply:
Let's continue the discussion in the other thread: https://github.com/microcharts-dotnet/Microcharts/pull/326#discussion_r1093778472 and can circle back.

Reply:
@Eilon: Is this answer enough to continue, or is there still something blocking?

Reply:
@Ghostbird once the other PR is merged I will rebase this PR (and remove the duplicated parts) so that this PR can go in too.

Reply:
@Eilon Can I do something to help speed this along?

Reply:
Hi, I hope to push out an update this week. I had a machine problem last week but that's resolved now. I just need to make sure I didn't mess up my rebase.